### DISCUSSION TITLE
Get Lookup field value with Token {Page.CustomProperty} return object

### DISCUSSION THREAD

**Discussion by VincentBD-dev** [microsoft-search/pnp-modern-search#1777] [Q&A]
**Version used**
3

**Describe the bug**

On the SharePoint page library, I have a custom property called "Client" of lookup field type

Via the PnP Modern Search Result WebPart, in my KQL query, I try to filter the search results with the token of the current page ((which contains the "Client" column) using the token {Page.Client}
Except that when I do this, looking at the "POST" request, the token {Page.Client} seems to return an object, and so it is not taken into account.

I searched through the various posts, but I couldn't find any particular syntax for the lookup fields.

**To Reproduce**
Create a lookup field on a page named "Client"
Set up a search result WebPart in one of these pages that contains this new column
In the request, use the token {Page.Client} for filter

**Expected behavior**
Here, the result should be filtered by the value of the "Lookup" column

**Screenshots**
![image](https://user-images.githubusercontent.com/62843661/151774011-5f95bafa-4437-4ac2-bd83-47b19f6d96f8.png)
![image](https://user-images.githubusercontent.com/62843661/151774183-770bd873-7c89-4a6d-9f9d-4f2fc1c98065.png)

**Desktop (please complete the following information):**
 - Chrome

**Comment by wobba** [comment]
Try `Page.Client.LookupValue` as it references what is available in the object model.

**Reply by VincentBD-dev** [reply]
Thanks, I tried to add the LookupValue, but it returns empty.

![image](https://user-images.githubusercontent.com/62843661/151803693-b4387230-1d64-4152-93bd-af85bc93d4c3.png)

![image](https://user-images.githubusercontent.com/62843661/151803785-17b7e68a-f421-4cc6-ab89-042bb572f86a.png)

However, when I check the metadata of the page, "Client" is well filled in: 

![image](https://user-images.githubusercontent.com/62843661/151803935-46f93d4c-8841-428d-a929-71d83a9fc54d.png)

**Reply by wobba** [reply]
I would have to debug to test it.. what about `Page.Client.Value`?

**Reply by VincentBD-dev** [reply]
Same result, I tested several keywords (Value, rawValue...)
